.golangci.toml

```diff
@@ -60,7 +60,7 @@ enable = [
     # "nlreturn",
     "noctx",
     "nolintlint",
-    "paralleltest",
```

Comment:
Removed this because having tests run in parallel was more cumbersome than anything while working through the test suite, and don't noticeably speed things up.